.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+# Description
+
+<!-- describe the changes you PR introduces here -->
+
+# Issues
+
+<!-- link related issues here -->
+
+# Checklist
+
+<!-- take care of the default items before creating your PR and be prepared to add more items -->
+
+* [ ] I updated the version in Chart.yaml
+* [ ] I added to the docs and updated them using  `pre-commit run -a`
+* [ ] CI is currently green and this is ready for review
```

Comment:
How can we know if CI is green before opening the PR?

Reply:
By looking at the results of github actions on your fork (the tests run on each push).

Reply:
You can also create a PR as "Draft" and then the test results will be easy to find but the PR will not get auto-assigned to a CODEOWNER until you mark it as ready.